### PR TITLE
Makes it more difficult to meta heretics with kitchen knives

### PR DESCRIPTION
**Pull request by Xoxeyos**
# General Documentation

### Intent of your Pull Request

This adds a shank that has slightly better stats than a glass shard, it is a subtype of the kitchen knife, so it will work with transmutations. No longer do you have to dive behind the cook's table, or scrounge for lathe parts. You can create a shank with ten cables, a glass shard, and a metal rod.

### Why is this change good for the game?

Heretics aren't forced to use standard kitchen knives anymore.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
This will be the basis of the Wiki entry for your PR, and more information / detail is better for Wiki editors to integrate.

Heretics can now use a shank to craft blades for their path chosen.

### What should players be aware of when it comes to the changes your PR is implementing?

A new weapon has been added, and it can be used to create heretic blades.

### What general grouping does this PR fall under? 

Anti-metagaming, a new weapon.

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
For example, "This new chem will heal X brute damage". 

It's a slightly better than the glass shard, with all the same stat types, embed, etc.

# Changelog

You can now create shanks using a shard of glass, a metal rod, ten cables, a welder, and a wire cutter.

:cl:  Xoxeyos
rscadd: Adds a shank to the game, as a kitchen knife subtype.
imageadd: Adds inhand shank sprites and icons for holding them.
/:cl:

### CODE REVIEW

**Comment by maxion12345** [comment]
who the hell sees an assistant stealing a kitchen knife and thinks "aw mate 'dats a fuckin 'eretic or me mum's name aint betsy bruv"

**Comment by Marmio64** [comment]
welder seems a bit unnecessary but eh

**Comment by ChesterTheCheesy** [comment]
I'd prefer if you could make it without tools (you don't need any for a spear), so you could make a shank in perma.

**Review by nesisknesis — COMMENTED** [review]
'icons/mob/inhands/weapons/items_righthand.dmi' and 
'icons/mob/inhands/weapons/items_lefthand.dmi' are doing the funky

i don't know a solution

image

**Comment by Xoxeyos** [comment]
> 
> 
> 'icons/mob/inhands/weapons/items_righthand.dmi' and
> 'icons/mob/inhands/weapons/items_lefthand.dmi' are doing the funky
> 
> i don't know a solution
> 
> image

I'm not sure why these were changed, and I don't know how they were.  Nothing should be changed with them.

**Comment by Xoxeyos** [comment]
So I think I went full retard at some point, and accidentally copied items_lefthand into inhands/weapons/ from /inhands/ at some point.

**Review by SomeguyManperson — APPROVED** [review]

**Review by nesisknesis — APPROVED** [review]

**Comment by MegaEmpirical** [comment]
Not needing tools makes sense, plus then you could make it in perma which I miss being able to do.


PS: I want to shank a bitch.
